static/vendor/trezor-connect.js

```diff
@@ -20168,7 +20168,7 @@ function __classPrivateFieldIn(state, receiver) {
 /************************************************************************/
 /******/ 	// The module cache
 /******/ 	var __webpack_module_cache__ = {};
-/******/ 	
+/******/
```

Comment:
please revert all the changes on this file

Reply:
Reverted